applications/serial_lte_modem/prj.conf

```diff
@@ -45,6 +45,7 @@ CONFIG_NRFX_TIMER2=y
 CONFIG_MAIN_STACK_SIZE=4096
 CONFIG_HEAP_MEM_POOL_SIZE=16384
 CONFIG_SYSTEM_WORKQUEUE_STACK_SIZE=4096
+CONFIG_AT_MONITOR_HEAP_SIZE=4096
```

Comment:
@lemrey I tested 512 and see all MCELL works, but 4096 should be even more secure